store/src/test/java/com/nytimes/android/external/store3/ClearStoreTest.kt

```diff
@@ -0,0 +1,93 @@
+package com.nytimes.android.external.store3
+
+import com.nytimes.android.external.store3.base.impl.BarCode
+import com.nytimes.android.external.store3.base.impl.Store
+import com.nytimes.android.external.store3.base.impl.StoreBuilder
+import io.reactivex.Maybe
+import io.reactivex.Single
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.Mockito.`when`
+import org.mockito.Mockito.verify
+import org.mockito.runners.MockitoJUnitRunner
+import java.util.concurrent.atomic.AtomicInteger
+
+@RunWith(MockitoJUnitRunner::class)
+class ClearStoreTest {
+    @Mock
+    lateinit var persister: ClearingPersister
+    lateinit var networkCalls: AtomicInteger
```

Comment:
`var`s that are not annotated with `@Mock` can be private! I think `@Mock` ones need to be publicly visibly, in order for the reflective initialization to work, other than that the rest can have limited visibility :)

Reply:
done!